### ISSUE TITLE
Callback?

### ISSUE THREAD

**Issue by gerey73** [mynenik/kForth-64#8]
When working with (gtk, mosquitto, ...), you often need a word for call back. Is it possible to implement similar in kForth?

http://gforth.org/manual/Callbacks.html#Callbacks

**Comment by mynenik** [comment]
Yes, it is possible to implement though it is not presently a feature of kForth. Callbacks to a Forth word are demonstrated in my project XYPLOT-32 . See the functions ForthCB() and ExecuteForthExpression() in the source file xyplot.cpp. The code illustrates use of a C++ callback function which uses callback data to compile and execute a Forth expression. It is written for Motif/X, but the principle should be the same under GTK, etc.
 
[https://github.com/mynenik/XYPLOT-32/blob/master/src/xyplot.cpp](url)
